Userland/Utilities/sed.cpp

```diff
@@ -90,6 +90,8 @@ static ErrorOr<SubstitutionCommand> parse_command(StringView command)
         } else if (flag == "g"sv) {
             // Allow multiple matches per line
             options &= ~PosixFlags::SingleMatch;
+        } else if (flag == "i"sv || flag == "I"sv) {
```

Comment:
so the 'g' flag is case sensitive, but the case insensitivity flag is case insensitive? 🤔 What does the spec say?

Reply:
The I/i flag is a GNU extension, so it's not in the POSIX spec. This behavior (g is case sensitive, but i is not) is how GNU describes it, and it's exactly how the default MacOS sed works. 

https://www.gnu.org/software/sed/manual/sed.html#The-_0022s_0022-Command